proposals/20200925-admins.md

```diff
@@ -0,0 +1,30 @@
+# Falco Infra Admins
+
+This document defines the roles and responsibilities of the Falco Infra Admins.
+
+## Terms & Definitions
+
+- 'the systems' -  an umbrella term refering to the collection of infrastructure and software as a service tools used by the Falco maintainers to develop, build, test, release, and distribute Falco.
+- Falco maintainers - the group of people listed in the OWNERS file in falcosecurity/falco
+- CNCF - The Cloud Native Computing Foundation
+- LF - The Linux Foundation
+
+## Motivation
+
+As part of modern open development, the Falco project is dependent on a considerable amount of development infrastructure. CI systems, code hosting systems, community websites, self-run infrastructure, and cloud services are all examples. Whenever possible, the use of role based access control is used to give the correct people access to the correct services, however in many cases a service account is used with a shared username and password. The group needing access and the number of systems to access are both growing and putting some policies in place will help the community scale effectively.
+
+## Goals
+- Define who is an "infra admin"
+- Define the responsibilities of an "infra admin"
+
+
+## Proposal
+
+- Infra admins are the individuals listed in the OWNERS file of falcosecurity/falco.
+- All systems involved in the development of Falco need to be managed by the infra admins.
+- The infra admins team will set up a process and tooling for secrets management. All secrets (passwords, tokens, etc) will be kept in this management system.
```

Comment:
Maybe worth adding or discussing in the next Falco Infra WG call some solutions, so to add the choice hereafter.

Reply:
Right. My thought was picking secrets management might slow this PR down so I just wrote that we should do it in some way.

Reply:
During the last meeting, we ended up deciding (and applying) for 1Password opensource account.

More in the meeting notes :)